opencga-app/app/scripts/azure/arm/mongodb/postdeploy.sh

```diff
@@ -31,12 +31,47 @@ installDeps () {
     sleep 15
 }
 
+
+scanForNewDisks() {
+    # Looks for unpartitioned disks
+    DEVS=($(fdisk -l 2>/dev/null | egrep -o '(/dev/[^:]*):' | egrep -o '([^:]*)'))
+
+    for DEV in "${DEVS[@]}";
+    do
+        isPartitioned ${DEV}
+    done
+}
+
+isPartitioned() {
+    OUTPUT=$(partx -s ${1} 2>&1)
+    if [[ $OUTPUT == *"failed to read partition table"* ]]; then
```

Comment:
should we quote `"$OUTPUT"` in case the path includes whitespace or empty?

Reply:
Fixed in https://github.com/opencb/opencga/pull/1109/commits/8f4f7ec0150c82a257beaee174ec332c214f4db8